examples/Serving-An-XGboost-Model-With-Merlin-Systems.ipynb

```diff
@@ -430,7 +430,7 @@
     "import tritonclient.grpc as grpcclient\n",
     "\n",
     "ten_examples = train.compute().drop(columns=['rating', 'title', 'rating_binary'])[:10]\n",
-    "inputs = convert_df_to_triton_input(inf_workflow.input_schema.column_names, ten_examples, grpcclient.InferInput)\n",
```

Comment:
The change to the signature of `convert_df_to_triton_input` will also affect the downstream notebooks in the Merlin repo.

Reply:
is the change only replacing this line with `inputs = convert_df_to_triton_input(workflow.input_schema, batch, grpcclient.InferInput)\n`?